warehouse/manage/forms.py

```diff
@@ -14,6 +14,8 @@
 
 import wtforms
 
+from urllib3 import PoolManager, Timeout
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏we use `requests` in other places in the code ([example](https://github.com/pypi/warehouse/blob/d502d5e233b7e51fad7bb161dc01d99455fec5c6/warehouse/oidc/forms/github.py#L71)), and this appears to be the first time we're using urllib3 directly. Is that to get both connect and read timeouts? If so, then we can pass those as a tuple to requests like so:

```
response = requests.head(field.data, timeout=(1.0, 1.0), headers=...)
```

But that apparently has the same affect as only specifying it once, since th value will be applied to both, so we could satisfy with

```
response = requests.head(field.data, timeout=1.0, headers=...)
```
https://requests.readthedocs.io/en/latest/user/advanced/#timeouts

I'm also wondering why we aren't using `request.http.head(...)` from https://github.com/pypi/warehouse/blob/1fbb4ac752e68b5840b9e09b68e44a165569bfa6/warehouse/http.py#L48-L53 more - is it that the forms don't have the `request` object available to them?